test/controllers/api/stories_controller_test.rb

```diff
@@ -16,9 +16,40 @@
 
     it 'can create a new story' do
       story_hash = { title: 'create story', set_account_uri: "/api/v1/accounts/#{account.id}" }
+      @request.env["CONTENT_TYPE"] = "application/json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.